index.html

```diff
@@ -0,0 +1,83 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@4.5.3/dist/css/bootstrap.min.css"/>
```

Comment:
Will our default be bootstrap for CSS?

Reply:
I doubt it. I just wanted to get running quickly for this PoC